sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/providers/BigQueryStorageWriteApiSchemaTransformProvider.java

```diff
@@ -371,7 +371,8 @@ public PCollectionRowTuple expand(PCollectionRowTuple input) {
               .setRowSchema(rowSchema);
 
       return PCollectionRowTuple.of(FAILED_ROWS_TAG, failedRowsOutput)
-          .and(FAILED_ROWS_WITH_ERRORS_TAG, failedRowsWithErrors);
+          .and(FAILED_ROWS_WITH_ERRORS_TAG, failedRowsWithErrors)
+          .and("errors", failedRowsWithErrors);
```

Comment:
we should probably not use a raw string here, and we should probably use a tupletag instead

Reply:
We can, but that should be high priority to clean up